.github/workflows/portal-loop.yml

```diff
@@ -46,3 +48,57 @@ jobs:
           push: ${{ github.event_name != 'pull_request' }}
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+
+  test-portal-loop-docker-compose:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v4
+
+      - name: "Setup the images"
+        run: |
+          cd misc/loop
+
+          docker compose build
+          docker compose pull
+          docker compose up -d
+
+      - name: "Test1 - Portal loop start gnoland"
+        run: |
+          while
+            block_height=$(curl -s localhost:26657/status | jq -r '.result.sync_info.latest_block_height')
+            echo "Current block height: $block_height"
+            [[ "$block_height" -lt 10 ]]
+          do
+            sleep 1
+          done
+
+          curl -s localhost:26657/status | jq
```

Comment:
this can be done inside the docker compose file as `healthcheck`

Reply:
This line is just to log some output.

Healthcheck is just for checking health, i want to ensure here and lower in the file that we're less than 10, to ensure we started again from scratch.